pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -180,6 +193,7 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 	app.Status.Services = appCompStatus
 	app.Status.SetConditions(readyCondition("HealthCheck"))
+	r.Recorder.Event(app, corev1.EventTypeNormal, ReasonHealthCheck, MessageHealthCheck)
 	app.Status.Phase = common.ApplicationRunning
 	err = handler.garbageCollection(ctx)
```

Comment:
please also add a final event after the GC

Reply:
ok